README.md

```diff
@@ -28,12 +28,13 @@ Crowdsourced list of Tech Communities separated by major cities in Canada
 | [Cloud Native Community Group - Toronto](https://community.cncf.io/toronto/) | CNCF, Cloud, K8s, Docker | [Discover Cloud Native Community Groups](https://community.cncf.io/) 
 | [Microsoft Reactor Toronto](https://www.meetup.com/microsoft-reactor-toronto/) | Networking, Workshops, Skills, Events | [Web](https://developer.microsoft.com/en-us/reactor/) [Meetup](https://www.meetup.com/microsoft-reactor-toronto/)
 | [HashiCorp User Group - Toronto](https://www.meetup.com/toronto-hashicorp-user-group/) | Cloud, IaC, Security | [Meetup](https://www.meetup.com/toronto-hashicorp-user-group/)
+| [K8SUG Canada](https://www.meetup.com/k8s-ca/) | Kubernetes, OpenShift, Cloud, On-prem | [Meetup](https://www.meetup.com/k8s-ca/) [LinkedIn](https://www.linkedin.com/company/k8sug/)
+| [Nepalese Tech Community in Canada](https://www.facebook.com/groups/nepalesetechcommunityincanada) | Technology, Networking, Mentorship | [Discord](https://discord.gg/kZ54z5vm).
```

Comment:
Can we remove the '.' at the end of the line? Also, can we add a Facebook hyperlink? 
| [Nepalese Tech Community in Canada](https://www.facebook.com/groups/nepalesetechcommunityincanada) | Technology, Networking, Mentorship | [Discord](https://discord.gg/kZ54z5vm) [Facebook](https://www.facebook.com/groups/nepalesetechcommunityincanada)

Reply:
Done.